lib/screens/landing/landing_screen.dart

```diff
@@ -1,14 +1,55 @@
 import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
+import 'package:survey/gen/assets.gen.dart';
 import 'package:survey/screens/screen.dart';
 
-class LandingScreen extends StatelessWidget {
+class LandingScreen extends StatefulWidget {
+  @override
+  _LandingScreenState createState() => _LandingScreenState();
+}
+
+class _LandingScreenState extends State<LandingScreen>
+    with TickerProviderStateMixin {
+  late final AnimationController _controller = AnimationController(
+    duration: const Duration(milliseconds: 800),
+    vsync: this,
+  );
+
+  late final Animation<double> _animation = CurvedAnimation(
+    parent: _controller,
+    curve: Curves.easeIn,
+  );
+
+  @override
+  void initState() {
+    super.initState();
+    _controller.forward();
+  }
+
   @override
   Widget build(BuildContext context) {
     return Screen(
-      body: Center(
-        child: Text("Landing"),
-      ),
-    );
+        body: Stack(
+      fit: StackFit.expand,
+      children: [
+        Container(
+          color: Colors.black,
+        ),
+        Assets.images.background.image(fit: BoxFit.fill),
```

Comment:
could you please declare a clearer name for `background` image? Its name's so general meaning 🤷‍♂️